Dockerfile

```
@@ -3,11 +3,15 @@
 #   $ docker run -v $(pwd):/app -ti csd rake test
 
-FROM --platform=linux/amd64 jetthoughts/cimg-ruby:3.4-chrome
+FROM jetthoughts/cimg-ruby:3.4-chrome
 
-# Install dependencies and clean up in one layer to reduce image size
-RUN sudo apt-get update -qq && \
-    DEBIAN_FRONTEND=noninteractive sudo apt-get install -qq \
+ENV DEBIAN_FRONTEND=noninteractive \
+ BUNDLE_PATH=/bundle
+
+RUN --mount=type=cache,target=/var/cache/apt \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)